cpp/ScanChain_Hgg.C

```diff
@@ -722,9 +724,13 @@ int ScanChain_Hgg(TChain *ch, double genEventSumw, TString year, TString process
       Electrons electrons = getElectrons(selectedPhotons);
       Muons muons = getMuons(selectedPhotons);
       //inverted, want events with leptons
-      //if ((electrons.size()) == 0) continue;
-      if (electrons.size() != 0 ) continue; 
-      if (muons.size() != 0 ) continue; 
+      if ( enrichDY ) {
+        if (electrons.size() == 0) continue;
+      }
+      else {
+        if (electrons.size() != 0 ) continue; 
+        if (muons.size() != 0 ) continue;
```

Comment:
Do we want to keep the muon veto only in the case of nominal cuts? Before this commit, I think it was applied also in the inverted-vetoes case. I think this is the way it should be, given that we want to only fiddle with the photon/electron selection.